src/dent.rs

```diff
@@ -177,18 +177,6 @@ impl DirEntry {
     }
 
     /// Returns true if and only if this entry points to a directory.
-    ///
-    /// This works around a bug in Rust's standard library:
-    /// https://github.com/rust-lang/rust/issues/46484
-    #[cfg(windows)]
-    pub(crate) fn is_dir(&self) -> bool {
-        use std::os::windows::fs::MetadataExt;
-        use winapi::um::winnt::FILE_ATTRIBUTE_DIRECTORY;
-        self.metadata.file_attributes() & FILE_ATTRIBUTE_DIRECTORY != 0
-    }
```

Comment:
Actually, does removing this let us get rid of `winnt` from the `winapi` dependency? https://github.com/BurntSushi/walkdir/blob/abf3a15887758e0af54ebca827c7b6f8b311cb45/Cargo.toml#L28

Reply:
You are correct. Actually the entire winapi dependency can be removed.

Reply:
Beautiful.

Reply:
Just curious, do you remember the reason for adding "std" feature in the first place?

Reply:
No idea. Probably because `walkdir` depends on std? Not sure. If CI works without `winapi`, then that's good enough for me.